README.Rmd

```diff
@@ -30,67 +30,109 @@ license](https://img.shields.io/badge/License-MIT-blue.svg)](https://github.com/
 
 ## Summary
 
-Tools to enable flexible and efficient hierarchical nowcasting of right-truncated epidemiological time-series using a semi-mechanistic Bayesian model with support for a range of reporting and generative processes. Nowcasting, in this context, is gaining situational awareness using currently available observations and the reporting patterns of historical observations. This can be useful when tracking the spread of infectious disease in real-time: without nowcasting, changes in trends can be obfuscated by partial reporting or their detection may be delayed due to the use of simpler methods like truncation. While the package has been designed with epidemiological applications in mind, it could be applied to any set of right-truncated time-series count data.
+_Nowcasting_ is predicting the *current* situation, addressing the practical reality that the data we want *now* will not be available until *later*. The `{epinowcast}` package provides you tools for nowcasting epidemiological data in a convenient, flexible, and efficient collection of `R` functions. Those functions implement a model of infectious disease spread and reporting using a semi-mechanistic Bayesian approach with a range of reporting and generative processes. Using this approach, you will typically see changes in trends earlier and more reliably than if using methods like truncation.
```

Comment:
big [citation needed] energy on that truncation bit at the end. i don't doubt its true, but readers definitely should.

Reply:
Hmm yes agree. As this first paragraph is also the package description can we move this edit to a new issue and discuss as its own PR?

Reply:
(note I quite like the changes but think it needs wider discussion/consideration)

Reply:
reverted in this, extracted as issue #393.